test/test-service-external-db.js

```diff
@@ -19,7 +18,7 @@ test('Service environment variable manipulation', function(t) {
     server.start(function onStart(err) {
       tt.ifError(err, 'server started successfully');
       Service = server.models.ServerService;
-      tt.equal(Service.dataSource, db);
+      tt.equal(Service.dataSource.connector.name, 'sqlite3');
```

Comment:
strong-mesh-models will use the memory connector if one isn't given. My concern here is that in the future if the default changes to sqlite3 and the feature for allowing an override breaks, this test will still pass and we won't notice the override is broken.